lib/services/local/usertoken.go

```diff
@@ -145,6 +145,6 @@ func (s *IdentityService) UpsertResetPasswordTokenSecrets(ctx context.Context, s
 }
 
 const (
-	passwordTokensPrefix = "resetpasswordtokens"
-	secretsPrefix        = "secrets"
+	userTokenPrefix = "usertoken"
```

Comment:
Doesn't this change have the potential to break in-flight reset token operations?

Reply:
yes, you'll get `invalid token` screen when user clicks on an old link. i thought about this but these tokens are pretty short-lived with defaults, resets are like 1hr to max 48hr, and invites are 8hr to max 24hr. And these tokens are pretty easy to re-generate for user, so I didn't think it'd be too much of an issue to re-create one (and old tokens will get eventually deleted since we set expiration in backend). Do you think this is too much an inconvenience for user?

Reply:
I could for a while, also check for the old prefix as well

Reply:
let's add a check for the old prefix when reading from the new prefix returns a `NotFound` error
it should be a small-ish change and it mostly removes this possible point of confusion

Reply:
revised to place checks for old prefix